utilities_common/cli.py

```diff
@@ -465,6 +465,13 @@ def run_command_in_alias_mode(command):
                 if "Vlan" in output:
                     output = output.replace('Vlan', '  Vlan')
                 print_output_in_alias_mode(output, index)
+            elif command.startswith("sudo ipintutil"):
+                "show ip(v6) int"
```

Comment:
Is this supposed to be a docstring? If so, should use triple-quotes.

Reply:
Fixed in latest commit